### ISSUE TITLE
Can't install both amphp/amp and amphp/react-adapter

### ISSUE THREAD

**Issue by ghost** [amphp/react-adapter#7]
```
/opt/project $ composer require amphp/react-adapter amphp/amp
Using version ^1.1 for amphp/react-adapter
Using version ^2.0 for amphp/amp
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for amphp/react-adapter ^1.1 -> satisfiable by amphp/react-adapter[v1.1.0].
    - Conclusion: remove react/event-loop v0.5.2
    - amphp/react-adapter v1.1.0 requires react/event-loop ^0.4.2 -> satisfiable by react/event-loop[v0.4.2, v0.4.3].
    - Can only install one of: react/event-loop[v0.5.2, v0.4.2].
    - Can only install one of: react/event-loop[v0.5.2, v0.4.3].
    - Installation request for react/event-loop (installed at v0.5.2) -> satisfiable by react/event-loop[v0.5.2].


Installation failed, reverting ./composer.json to its original content.
```

This will happen even if no other packages are required!

**Comment by kelunik** [comment]
This isn't an issue with `amphp/amp`, but rather `react/event-loop`. I have an update locally that makes this package compatible with `react/event-loop` v0.5.x, but I haven't pushed those changes, yet. See also #6.

**Comment by prolic** [comment]
v1.0 of react-event-loop is out

**Comment by zek** [comment]
Is there any plan to upgrade this library?

**Comment by kelunik** [comment]
Yes, there is. The latest master is already compatible, I think. Not all tests are passing, yet.

**Comment by zek** [comment]
react/event-loop released v1.0.0 so I cant install due to version conflict

**Comment by kelunik** [comment]
Most React libraries also work with older versions of their event loop, at least that used to be the case. I'll try to have a look at this issue in the coming week.

**Comment by kelunik** [comment]
The new [v2.0.0 release](https://github.com/amphp/react-adapter/releases/tag/v2.0.0) has been published and is compatible with 0.5 and 1.0.

**Comment by zek** [comment]
👍
